ArduPlane/Log.cpp

```diff
@@ -234,7 +234,7 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: ThrOut: scaled output throttle
 // @Field: RdrOut: scaled output rudder
 // @Field: ThrDem: demanded speed-height-controller throttle
-// @Field: Aspd: airspeed estimate
+// @Field: Aspd: airspeed estimate (measurement if airspeed sensor present and in use)
```

Comment:
```suggestion
// @Field: Aspd: airspeed estimate (or measurement if airspeed sensor present and healthy)
```

Reply:
Except the measurement is reported only if ARSPD_USE is non-zero
regardless of whether  the sensor is healthy.  

Pushed a change to resolve that:
(or measurement if airspeed sensor healthy and ARSPD_USE>0)

Of course the function which returns (_airspeed->use() && _airspeed->healthy()) is named "airspeed_sensor_enabled" :)

Since function AP_AHRS_DCM::airspeed_estimate() may return either an estimate or a "measurement" I assume the original author viewed pitot airspeed as an "estimate" calculated from a "measurement" of differential pressure. To maintain the distinction between estimate and measurement I suppose we would need 2 separate fields plus another to indicate which is in use.